web-js/src/hooks/useCollection.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { useRealmApp } from "../components/RealmApp";
+
+/**
+ * Returns a MongoDB Collection client object
+ * @template DocType extends Realm.Services.MongoDB
+ * @param {Object} config - A description of the collection.
+ * @param {string} [config.service="mongodb-atlas"] - The service name of the collection's linked cluster.
```

Comment:
What else could this be?  Is it possible to link this to another service?

Reply:
It's possible to link a MongoDB Realm app to multiple MongoDB Altas clusers as "data sources" and it's therefore possible for a user to define the service name. "mongodb-atlas" is just the default.